.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,25 @@
+name: algolia-scraper
```

Comment:
This should not be in here, since it's happening in #62.  Please remove all the algolia/search stuff from this PR and only do your deployment work.

Reply:
@trangntt-016 ^

Reply:
deleted it